src/RepairManagerAgent/main.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import time
+import yaml
+import logging
+import logging.config
+import os
+
+with open('./logging.yaml', 'r') as log_file:
+    log_config = yaml.safe_load(log_file)
+
+logging.config.dictConfig(log_config)
+logger = logging.getLogger(__name__)
+
+def reboot_node():
+    os.system('sync')
+    os.system('reboot -f')
+
+def Run():
+    while True:
+        etcd_server = os.environ["ETCD_SERVER"]
+        etcd_port = os.environ["ETCD_PORT"]
+        node_name = os.environ["NODE_NAME"]
+        etcd_url = f'http://{etcd_server}:{etcd_port}/v2/keys/{node_name}/reboot'
```

Comment:
the key is {NODE_NAME}/reboot, so each pod on each worker node will be polling for a different key

Reply:
more on etcd data model: https://etcd.io/docs/v3.4.0/learning/data_model/